pkg/operator/ceph/pool/controller.go

```diff
@@ -80,6 +80,11 @@ func (c *PoolController) StartWatch(namespace string, stopCh chan struct{}) erro
 }
 
 func (c *PoolController) onAdd(obj interface{}) {
+	if c.clusterSpec.External.Enable && !c.clusterSpec.External.RunStatelessWorkload {
+		logger.Warningf("Creating pools for an external ceph cluster is disabled (RunStatelessWorkload is false)")
```

Comment:
Nit: from the user-facing side of things, this is `runStatelessWorkload` not capitalized.